services/web-app/components/hero/hero.module.scss

```diff
@@ -46,15 +56,20 @@ $hero-height: convert.rem(560px);
 }
 
 /*
-  * Assets hero image & custom styles
+  * Homepage hero image & custom styles
   */
 
-.assets::before {
-  background: url(../../pages/index/images/assets-hero_mobile.png) no-repeat bottom right / 100%
-    auto;
+.homepage::before {
+  background: theme.$background url(../../pages/index/images/assets-hero_mobile.png) no-repeat
+    bottom center / auto $hero-height-sm;
 
   @include breakpoint.breakpoint(md) {
-    background: url(../../pages/index/images/assets-hero.png) no-repeat top right / auto
-      $hero-height;
+    background: theme.$background url(../../pages/index/images/assets-hero.png) no-repeat top right /
+      auto $hero-height-sm;
+  }
+
+  @include breakpoint.breakpoint(lg) {
+    background: theme.$background url(../../pages/index/images/assets-hero.png) no-repeat top right /
```

Comment:
image is on top of text at lg breakpoint 
<img width="967" alt="image" src="https://user-images.githubusercontent.com/40550942/185989386-22a2e580-3cc7-499c-b36a-74dcd319d848.png">

Reply:
Just added the latest images from Jeannie, there is a slight overlap right before it switches from lg to medium, but should be ok.